.gitignore

```diff
@@ -35,4 +35,11 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 
-*storybook.log
\ No newline at end of file
+#playwright
+/test-results/
+/playwright-report/
```

Comment:
Is this something we want to ignore? We will eventually create a deployment that can output the Playwright report so the person creating the PR can see this.

Reply:
@shashilo Thanks for explaining, Shashi. I will remove from the .gitignore file.